pkg/controller/backup/backup_controller.go

```diff
@@ -173,6 +174,9 @@ func (bkc *Controller) sync(key string) error {
 		return err
 	}
 
+	if !validBackup(backup) {
+		return nil
+	}
```

Comment:
We can define a BackupInvalid [BackupConditionType](https://github.com/pingcap/tidb-operator/blob/master/pkg/apis/pingcap/v1alpha1/types.go#L622), if check backup failed, update this condition. Then make a judgment [here](https://github.com/pingcap/tidb-operator/blob/master/pkg/controller/backup/backup_controller.go#L194), so that the invalid backup CR will not be enqueued.

Reply:
updated